src/models/service.ts

```diff
@@ -35,10 +36,51 @@ import { MaxAllowedPaymentAmount } from "../../generated/definitions/MaxAllowedP
 export const SERVICE_COLLECTION_NAME = "services";
 export const SERVICE_MODEL_PK_FIELD = "serviceId";
 
+export enum ScopeEnum {
+  "NATIONAL" = "NATIONAL",
+
+  "LOCAL" = "LOCAL"
+}
+
+// required attributes
+const ServiceMetadataR = t.interface({
+  scope: enumType<ScopeEnum>(ScopeEnum, "scope")
+});
+
+// optional attributes
+const ServiceMetadataO = t.partial({
+  description: NonEmptyString,
+
+  webUrl: NonEmptyString,
```

Comment:
@gunzip I guess we prefer the snake_case, is it so?
 Indeed we defined that fields in snake_case in [io-service-metadata](https://github.com/pagopa/io-services-metadata/blob/master/definitions/Service.ts)

Reply:
this is the internal model used by Typescript and CosmosDB. The conversion in snake case happens here:https://github.com/pagopa/io-functions-admin/blob/master/utils/conversions.ts